src/themes/teams/components/List/listItemStyles.ts

```diff
@@ -2,16 +2,32 @@ import { pxToRem } from '../../../../lib'
 import { IComponentPartStylesInput, ICSSInJSStyle } from '../../../../../types/theme'
 import { IListItemProps } from '../../../../components/List/ListItem'
 
+const hoverStyle = variables => ({
+  background: variables.selectionHoverBackgroundColor,
+  color: variables.selectionHoverColor,
+  cursor: 'pointer',
+
+  '& .ui-item-layout__header': { color: 'inherit' },
```

Comment:
this will be rendered by Fela as the following selector
```css
.a .ui-item-layout__header
```

where `.a` is class of `ListItem`. The problem I see here is that these styles will be applied to all `.ui-item-layout__header` components that are **_anywhere_** in the child tree of `ListItem` - although our intent was to provide these styles for direct `.ui-item-layout__header` child only.

Reply:
We could use '>' selector instead of ' '. I will check that later today.

Reply:
direct child selector, unfortunately, won't work in this case, because rendered DOM tree is like the following for the list item:
```html
<div .ui-list__item>
  ..
  <div .ui-layout__main>  // this one prevents direct child selector to work
     ...
     <div .ui-item-layout__header>
```

Reply:
As discussed offline - "we are not making things any worse"